lua/minigames/engine/sh_convars.lua

```diff
@@ -43,6 +43,13 @@ if SERVER then
 			local mg = mgs[i]
 			local cvd = mg.conVarData
 
+
+			local mg_enable = CreateConVar("ttt2_minigames_" .. mg.name .. "_enabled", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE, FCVAR_REPLICATED})
```

Comment:
Here, this cvar is already created: https://github.com/TTT-2/ttt2mg/pull/4/files#diff-01b82fcce9cc361047c79fe0ec189315R93. This seems to be inconsistent. Because SetupData was made to setup minigames related data, this should be kept there. As a result, `GetConVar` should be used here.

Reply:
For this, I was finding that I was running into errors because sh_convar was being initialized before the SetupData, resulting in the convars not existing yet

Reply:
If that's the case, SetupData is useless (remove) or in the wrong loading direction (correct).